### DISCUSSION TITLE
Create a new Deta micros under a specific project Error

### DISCUSSION THREAD

**Discussion by tongsangcong** [deta/cloud-docs#358] [Help] [answered]
Hello, I have followed the Deta docs on creating new micros under a specific project, but I ended up with Error "Error: failed to create new program: No such project"

what I type is below (following the docs):
`deta new --node --project init-exist`

actually I want to create a data micros so I could deploy my svelte-kit project to it (SSR it via data micros)

**Comment by abdelhai** [comment, marked as answer]
hi, does the project "init-exist" already exists?

if not, you need to create a project (click on the arrow next to the logo)
![image](https://user-images.githubusercontent.com/1752577/182019685-6a583381-d898-4d4b-9270-d31c7e20452d.png)

**Reply by tongsangcong** [reply]
wow, amazing, thank you
